tailscale/rootfs/etc/s6-overlay/scripts/stage2_hook.sh

```diff
@@ -25,3 +25,8 @@ fi
 if bashio::config.false 'taildrop'; then
     rm /etc/s6-overlay/s6-rc.d/user/contents.d/taildrop
 fi
+
+# Disable serve service when proxy and/or funnel has not been explicitly enabled
+if ! bashio::config.true 'proxy'; then
```

Comment:
I guess you need to check for proxy OR funnel = true if the service handles both